app/src/main/res/values-fr/strings.xml

```diff
@@ -26,15 +26,15 @@
     <string name="configAction_share">Partager</string>
     <string name="configAction_export">Exporter</string>
     <string name="configAction_import">Importer</string>
-    <string name="configAction_info">More Info</string>   <!-- TODO -->
-    <string name="configAction_sortBy">Sort by</string>                <!-- TODO -->
+    <string name="configAction_info">Plus d'info</string>   <!-- TODO -->
```

Comment:
There are a couple spots where you have a `' ` character that needs to be escaped with a slash; e.g.`\'`. 

Other than that these changes look fine. 👍 I can merge them whenever you feel they are ready.